views/partials/social_media_sharing_buttons.hbs

```diff
@@ -1,10 +1,12 @@
 <ul class="social-media-sharing-buttons">
   <li>
+    <a href="https://www.facebook.com/sharer/sharer.php?u={{ SERVER_URL}}" target="_blank" rel="noopener noreferrer">
     <img class="facebook-button" alt="Share Firefox Monitor on Facebook" src="/img/social_media_icons/social-facebook.svg" />
     Share
+    </a>
   </li>
   <li>
-    <a class="twitter-share-button" href="https://twitter.com/intent/tweet?text=Find%20out%20if%20your%20email%20addresses%20have%20been%20hacked%20at%20https://monitor.firefox.com.">
+    <a class="twitter-share-button" href="https://twitter.com/intent/tweet?text=Find%20out%20if%20your%20email%20addresses%20have%20been%20hacked%20at%20{{ SERVER_URL }}" target="_blank" rel="noopener noreferrer">
```

Comment:
Oooo, we could make this easier to read with an `encodeURI` handlebars helper:
`hbs-helpers.js`:
```js
     hbs.registerHelper("encodeURI", function(input) {
       return encodeURI(input);
     });
```
`views/partials/social_media_sharing_buttons.hbs`:
```html
<a class="twitter-share-button" href="https://twitter.com/intent/tweet?text={{encodeURI "Find out if your email addresses have been hacked at "}}{{ SERVER_URL }}" rel="noopener noreferrer">
```